src/Data/Fin/Permutation/Components.agda

```diff
@@ -19,8 +19,15 @@ open import Algebra.FunctionProperties using (Involutive)
 --  Functions
 --------------------------------------------------------------------------------
 
-swap : ∀ {n} → Fin n → Fin n → Fin n → Fin n
-swap i j k with k ≟ i
+-- 'tranpose i j' acts like the identity function, except it swaps the places of
+-- 'i' and 'j'.
+--
+-- swap i j i ≡ j
+-- swap i j j ≡ i
+-- swap i j k ≡ k if k ≠ i and k ≠ j
+
```

Comment:
Should the comment also be updated to use the new name?

Reply:
Done